### PR TITLE
docs: update powered-by file

### PR DESCRIPTION
**Pull request by guoqqqi**
### What this PR does / why we need it:

Updated by July 9, 2021 User logo, and sorted by English name A-Z
### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Comment by guoqqqi** [comment]
After sorting confirmation, the A-Z markers will be removed.
Remark.
- doggy.cool (link won't open)
- `zhuzheng` ? (Is this an individual or an organization)
- 小糖科技 (is it sugar beans?)
See 
https://github.com/apache/apisix/issues/487

File: powered-by.md [hunk 1]
```diff
@@ -26,50 +26,105 @@ or providing commercial products including APISIX.
...
-1. <a href="https://www.teamones.cn/" rel="nofollow">万思</a>
-1. <a href="http://travelsky.com" rel="nofollow"> Travelsky 中国航信</a>
-1. <a href="https://vbill.cn/" rel="nofollow">vbill 随行付</a>
-1. <a href="https://www.willclass.com/" rel="nofollow">willclass 会课</a>
-1. <a href="https://www.xin.com/" rel="nofollow">Xin 优信二手车</a>
-1. <a href="https://xueqiu.com/" rel="nofollow">雪球</a>
-1. <a href="https://www.zihao.biz/" rel="nofollow">紫豪网络</a>
-1. <a href="https://www.zuzuche.com/" rel="nofollow">zuzuche 租租车</a>
-1. <a href="https://www.zybang.com/" rel="nofollow">zybang 作业帮</a>
+A:
```
Comment (juzhiyuan) [inline powered-by.md:29]: No need to add this

File: powered-by.md [hunk 2]
```diff
@@ -26,50 +26,64 @@ or providing commercial products including APISIX.
...
+1. <a href="http://www.hys.cn/" rel="nofollow">好医生集团</a>
 1. <a href="https://www.ihomefnt.com/" rel="nofollow">ihomefnt 艾佳生活</a>
+1. <a href="https://en.intsig.com/" rel="nofollow">intsig 上海合合信息科技股份有限公司</a>
 1. <a href="https://www.jiandanxinli.com/" rel="nofollow">jiandanxinli 简单心理</a>
-1. <a href="https://www.ke.com/" rel="nofollow">ke.com 贝壳找房</a>
 1. <a href="https://jr.ly.com/" rel="nofollow">jr.ly 同程金服</a>
+1. <a href="https://www.ke.com/" rel="nofollow">ke.com 贝壳找房</a>
 1. <a href="https://www.meizu.com/" rel="nofollow">Meizu 魅族</a>
 1. <a href="https://www.mingyuanyun.com/" rel="nofollow">明源云客</a>
+1. <a href="https://www.meicai.cn/#/" rel="nofollow">美菜网</a>
```
Comment (juzhiyuan) [inline powered-by.md:61]:
```suggestion
1. <a href="https://www.meicai.cn/" rel="nofollow">美菜网</a>
```

File: powered-by.md [hunk 3]
```diff
@@ -26,50 +26,64 @@ or providing commercial products including APISIX.
...
+1. <a href="https://game.qq.com/" rel="nofollow">game.qq.com 腾讯游戏</a>
 1. <a href="http://haieruplus.com/" rel="nofollow">haieruplus 海尔优家</a>
 1. <a href="http://www.hellowin.cn/" rel="nofollow">hellowin 好洛维</a>
 1. <a href="https://www.hellotalk.com/" rel="nofollow">HelloTalk, Inc.</a>
 1. <a href="https://www.htzhiyun.cn/" rel="nofollow">航天网信</a>
 1. <a href="http://huawei.com/" rel="nofollow">Huawei 华为</a>
 1. <a href="https://www.huya.com/" rel="nofollow">虎牙</a>
+1. <a href="http://www.hys.cn/" rel="nofollow">好医生集团</a>
 1. <a href="https://www.ihomefnt.com/" rel="nofollow">ihomefnt 艾佳生活</a>
+1. <a href="https://en.intsig.com/" rel="nofollow">intsig 上海合合信息科技股份有限公司</a>
```
Comment (juzhiyuan) [inline powered-by.md:55]:
```suggestion
1. <a href="https://www.intsig.com/" rel="nofollow">intsig 上海合合信息科技股份有限公司</a>
```

File: powered-by.md [hunk 4]
```diff
@@ -26,50 +26,64 @@ or providing commercial products including APISIX.
...
 1. <a href="https://www.dataoke.com/" rel="nofollow">dataoke 大淘客</a>
-1. <a href="http://dian.so/" rel="nofollow">小电科技</a>
 1. <a href="https://www.didachuxing.com/" rel="nofollow">嘀嗒出行</a>
+1. <a href="http://dusto.cn/" rel="nofollow">dusto.cn 浙江大东鞋业有限公司</a>
+1. <a href="http://dian.so/" rel="nofollow">dian.so 小电科技</a>
 1. <a href="https://www.efactory-project.eu/" rel="nofollow">eFactory</a>
 1. <a href="https://www.ehomepay.com.cn/" rel="nofollow">ehomepay 理房通</a>
 1. <a href="https://ezone.work/" rel="nofollow">eZone 简单一点科技</a>
 1. <a href="https://fansup.mobi/" rel="nofollow">fansup</a>
+1. <a href="https://game.qq.com/" rel="nofollow">game.qq.com 腾讯游戏</a>
```
Comment (juzhiyuan) [inline powered-by.md:46]:
```suggestion
1. <a href="https://game.qq.com/" rel="nofollow">Tencent Game 腾讯游戏</a>
```

File: powered-by.md [hunk 5]
```diff
@@ -26,50 +26,64 @@ or providing commercial products including APISIX.
...
+1. <a href="https://www.chaolian360.com/" rel="nofollow">chaolian 超链云商</a>
+1. <a href="https://www.ccbft.com/" rel="nofollow">ccb fintech 建信金科</a>
+1. <a href="http://51tiangou.com/" rel="nofollow">51tiangou 大商天狗</a>
 1. <a href="https://www.daocloud.io/" rel="nofollow">DaoCloud</a>
 1. <a href="https://www.dasouche.com/" rel="nofollow">dasouche 大搜车</a>
 1. <a href="https://www.dataoke.com/" rel="nofollow">dataoke 大淘客</a>
-1. <a href="http://dian.so/" rel="nofollow">小电科技</a>
 1. <a href="https://www.didachuxing.com/" rel="nofollow">嘀嗒出行</a>
+1. <a href="http://dusto.cn/" rel="nofollow">dusto.cn 浙江大东鞋业有限公司</a>
+1. <a href="http://dian.so/" rel="nofollow">dian.so 小电科技</a>
```
Comment (juzhiyuan) [inline powered-by.md:41]:
```suggestion
1. <a href="http://dian.so/" rel="nofollow">Dian 小电科技</a>
```

File: powered-by.md [hunk 6]
```diff
@@ -26,50 +26,64 @@ or providing commercial products including APISIX.
...
-1. <a href="http://www.163.com" rel="nofollow">Netease 网易</a>
-1. <a href="https://www.360.cn/" rel="nofollow">360 奇虎</a>
-1. <a href="http://51tiangou.com/" rel="nofollow">51tiangou 大商天狗</a>
 1. <a href="http://aimiaobi.com/" rel="nofollow">aimiaobi 妙笔 AI</a>
-1. <a href="https://www.chaolian360.com/" rel="nofollow">chaolian 超链云商</a>
+1. <a href="http://www.augurit.com/" rel="nofollow">奥格科技股份有限公司</a>
+1. <a href="https://cloud.aispeech.com/" rel="nofollow">aispeech 思必驰信息科技股份有限公司</a>
 1. <a href="http://www.cunw.com.cn/" rel="nofollow">cunw 湖南新云网</a>
+1. <a href="https://www.chaolian360.com/" rel="nofollow">chaolian 超链云商</a>
+1. <a href="https://www.ccbft.com/" rel="nofollow">ccb fintech 建信金科</a>
```
Comment (juzhiyuan) [inline powered-by.md:34]:
```suggestion
1. <a href="https://www.ccbft.com/" rel="nofollow">CCB Fintech 建信金科</a>
```

File: powered-by.md [hunk 7]
```diff
@@ -26,50 +26,64 @@ or providing commercial products including APISIX.
...
-1. <a href="https://hy.10086.cn/" rel="nofollow">中移杭研</a>
-1. <a href="http://www.163.com" rel="nofollow">Netease 网易</a>
-1. <a href="https://www.360.cn/" rel="nofollow">360 奇虎</a>
-1. <a href="http://51tiangou.com/" rel="nofollow">51tiangou 大商天狗</a>
 1. <a href="http://aimiaobi.com/" rel="nofollow">aimiaobi 妙笔 AI</a>
-1. <a href="https://www.chaolian360.com/" rel="nofollow">chaolian 超链云商</a>
+1. <a href="http://www.augurit.com/" rel="nofollow">奥格科技股份有限公司</a>
+1. <a href="https://cloud.aispeech.com/" rel="nofollow">aispeech 思必驰信息科技股份有限公司</a>
 1. <a href="http://www.cunw.com.cn/" rel="nofollow">cunw 湖南新云网</a>
+1. <a href="https://www.chaolian360.com/" rel="nofollow">chaolian 超链云商</a>
```
Comment (juzhiyuan) [inline powered-by.md:33]:
```suggestion
1. <a href="https://www.chaolian360.com/" rel="nofollow">Chaolian 超链云商</a>
```

File: powered-by.md [hunk 8]
```diff
@@ -26,50 +26,64 @@ or providing commercial products including APISIX.
...
 Users are encouraged to add themselves to this page, [issue](https://github.com/apache/apisix/issues/487) and PR are welcomed.
 
-1. <a href="https://hy.10086.cn/" rel="nofollow">中移杭研</a>
-1. <a href="http://www.163.com" rel="nofollow">Netease 网易</a>
-1. <a href="https://www.360.cn/" rel="nofollow">360 奇虎</a>
-1. <a href="http://51tiangou.com/" rel="nofollow">51tiangou 大商天狗</a>
 1. <a href="http://aimiaobi.com/" rel="nofollow">aimiaobi 妙笔 AI</a>
-1. <a href="https://www.chaolian360.com/" rel="nofollow">chaolian 超链云商</a>
+1. <a href="http://www.augurit.com/" rel="nofollow">奥格科技股份有限公司</a>
+1. <a href="https://cloud.aispeech.com/" rel="nofollow">aispeech 思必驰信息科技股份有限公司</a>
```
Comment (juzhiyuan) [inline powered-by.md:31]:
```suggestion
1. <a href="https://cloud.aispeech.com/" rel="nofollow">AISPEECH 思必驰信息科技股份有限公司</a>
```

File: powered-by.md [hunk 9]
```diff
@@ -26,50 +26,64 @@ or providing commercial products including APISIX.
 
 Users are encouraged to add themselves to this page, [issue](https://github.com/apache/apisix/issues/487) and PR are welcomed.
 
-1. <a href="https://hy.10086.cn/" rel="nofollow">中移杭研</a>
-1. <a href="http://www.163.com" rel="nofollow">Netease 网易</a>
-1. <a href="https://www.360.cn/" rel="nofollow">360 奇虎</a>
-1. <a href="http://51tiangou.com/" rel="nofollow">51tiangou 大商天狗</a>
 1. <a href="http://aimiaobi.com/" rel="nofollow">aimiaobi 妙笔 AI</a>
-1. <a href="https://www.chaolian360.com/" rel="nofollow">chaolian 超链云商</a>
+1. <a href="http://www.augurit.com/" rel="nofollow">奥格科技股份有限公司</a>
```
Comment (juzhiyuan) [inline powered-by.md:30]:
```suggestion
1. <a href="http://www.augurit.com/" rel="nofollow">AUGUR 奥格科技股份有限公司</a>
```

**Review by juzhiyuan — APPROVED** [review]

**Review by membphis — APPROVED** [review]

**Comment by membphis** [comment]
We need to update this picture:

image

**Review by spacewander — APPROVED** [review]

**Comment by Yiyiyimu** [comment]
> We need to update this picture:
> 
> image

cc @guoqqqi

**Comment by juzhiyuan** [comment]
@iamayushdas Hi, do you have the Logo Wall project file?

**Comment by iamayushdas** [comment]
> @iamayushdas Hi, do you have the Logo Wall project file?

I am not sure if i still have that, but i could make one

**Comment by juzhiyuan** [comment]
🤔 because this image will get updated frequently, would better put a project file on Figma/PhotoShop or some other tools :)

**Comment by iamayushdas** [comment]
Oh yes I remember i have putted it in figma

**Comment by juzhiyuan** [comment]
That would be better, @Serendipity96 will need to select some logos, I will tell her to contact you :)

**Comment by iamayushdas** [comment]
> That would be better, @Serendipity96 will need to select some logos, I will tell her to contact you :)

okay sir

**Pull request merged**